README.md

```diff
@@ -9,16 +9,38 @@
 
 <img width="710" alt="math magicians preview" src="#">
 
-<h2>Built With</h2>
+<h2>🧩 Built With</h2>
 
 React.js
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+<h2>Live Demo</h2>
 
-In the project directory, you can run:
+<a href="#" rel="noopener noreferrer">Live Demo Link</a>
 
+<h2>📚 Getting Started</h2>
+
+To get a local copy up and running follow these simple example steps.
+
+<h2>Prerequisites</h2>
+
+For this project you don't need any aditional tools to run it.
+Be sure to have git set up on your local machine. If you are unfamiliar with the necessary steps, please find a step-by-step guide <a href="https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup" rel="noopener noreferrer">here</a>
+
+<h2>Setup</h2>
+
+Fork a copy to your repository and clone it locally
+
+<h2>Install</h2>
+
+Run `npm install` from your terminal window
+
+<h2>Usage</h2>
+
+You can now edit the code in your favorite Code Editor or view it in your browser.
+
+<h2>Deployment</h2>
 ### `npm start`
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to format this line better. You can probably get the desired result by leaving a line space between **`<h2>Deployment</h2>`** and **### npm start**